libraries/ui-library-angular/src/lib/link/six-router-link.directive.ts

```diff
@@ -0,0 +1,30 @@
+import { Attribute, Directive, ElementRef, OnChanges, Renderer2, SimpleChanges } from '@angular/core';
+import { ActivatedRoute, Router, RouterLink } from '@angular/router';
+import { LocationStrategy } from '@angular/common';
+
+@Directive({
+  selector: 'six-sidebar-item[routerLink],six-button[routerLink][type=link]',
```

Comment:
I would add the `href` attribute to all `six-button` types and not only to those of type `link`.